functions/logDonation.js

```diff
@@ -1,80 +1,42 @@
-const functions = require('firebase-functions');
```

Comment:
I removed un-used variables after removing the `logDonation` function.  Since that's depreciated and we don't require backwards compatibility I felt that it was safe to remove.